frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -27,6 +28,7 @@ export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pip
     unfilteredResources
       ?.filter(({ resourceRef }) => !!resourceRef)
       .map((resource) => resource.resourceRef.name) || [];
+  const pipelineRunStatus = pipelineRunFilterReducer(pipelineRun);
```

Comment:
```suggestion
  const pipelineRunStatus = pipelineRunStatus(pipelineRun);
```
That way you get a null-able value.